destination/config.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright © 2022 Meroxa, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package destination
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/conduitio/conduit-connector-zendesk/config"
+)
+
+const (
+	// KeyBufferSize is the config name for buffer size.
+	KeyBufferSize = "bufferSize"
+
+	// KeyMaxRetries is the number of times the writer needs to retry, in case of 429 error, before returning an error
+	KeyMaxRetries = "maxRetries"
+
+	// maxBufferSize determines maximum buffer size a config can accept.
+	// When config with bigger buffer size is parsed, an error is returned.
+	maxBufferSize uint64 = 100
```

Comment:
It would be good to add what Aman said [here](https://github.com/gopherslab/conduit-connector-zendesk/pull/2#discussion_r881910218) to the comment ^.

Reply:
commented the reason for the ceil limit. done